touringcars.map

```diff
@@ -14,6 +14,7 @@
 # naam                  datum         wijziging
 # ------------------    ----------    -----------------------------------------
 # Chris van Riel        29-06-2020    Initiatie
+# CHris van Riel        08-07-2020    Replaced field titel by omschrijving (new specs)
```

Comment:
Kijk nog even in symbols/symbollist.txt op regel 11 ;-)

Reply:
Aha ik zie het :-D in git dus ;-)

Reply:
Wordt trouwens verschillend over gedacht hoor. Veel open source projecten hebben een CHANGES.txt (soms weer automatisch obv. git commit messages). Wellicht interessant om eens in de groep te gooien hoe wij hiermee willen omgaan.